content/about/notes/1.0.1/index101.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Istio 1.0.1
+weight: 91
+icon: /img/notes.svg
+---
+
+We're proud to release Istio 1.0.1! Within this release, we have been addressing some critical issues found by the community when using Istio 1.0.
+
+These release notes describe what's different between Istio 1.0 and Istio 1.0.1. 
+
+## Networking
+- Pilot scability and Envoy sidecar startup improvement.
+- Fixed virtual service host mismatch issue when port is added.
+- Allow multiple destination rules with different subsets for the same host.
+- Ability to merge non conflict http routes in multiple virtual service resources that are bounded to the same gateway resource.
```

Comment:
Replace the above 2 bullets with the following:

* Added limited support for [merging multiple virtual service or destination rule definitions](/help/ops/traffic-management/deploy-guidelines/#multiple-virtual-services-and-destination-rules-for-the-same-host) for the same host.